content/authors/jibay-mcs.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Jibay Mcs
+slug: jibay-mcs
+avatar: jibay-mcs.jpg
+github_url: https://github.com/JibayMcs
+twitter_url: https://twitter.com/ZeAmateis
+---
+
+Just an old lumberjack, playing with Laravel
```

Comment:
Could you add a little bit more of a bio here, so it's similar to the others on our website?